packages/react/src/analytics.doc.ts

```diff
@@ -0,0 +1,42 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'sendShopifyAnalytics',
+  category: 'utilities',
+  isVisualComponent: false,
+  related: [],
```

Comment:
We will probably want to add to this `related` array as we add other analytics docs